PersonListener.Tests/E2ETests/Steps/PersonAddedToTenureUseCaseSteps.cs

```diff
@@ -29,7 +29,7 @@ public void GivenAMessageWithNoPersonAdded(TenureResponseObject tenure)
             var eventSns = CreateEvent(tenure.Id);
             var newData = tenure.HouseholdMembers;
             var oldData = newData.DeepClone();
-            eventSns.EventData = new EventData()
+            eventSns.EventData = new PersonListener.Boundary.EventData()
```

Comment:
can this `PersonListener.Boundary` not be retrieved from the using?

Reply:
There is an EventData object in the Person Shared package, however, it is slightly different to the one used in the Listener.
In the listener, `OldData` and `NewData` have types: `Dictionary<string, object>`, but in the API (and shared package), they are `object`s.

I set this up temporarily because I'm not sure which version is supposed to be used? I had to specify the PersonListener version so that it doesn't use the shared version, which causes errors.

Reply:
Makes sense, but can you not do something like `using EventData = PersonListener.Boundary` within the using section (so around line 10 ) and then over here you can just leave it as it is